Neos.Neos/Classes/AssetUsage/Projection/AssetUsageRepository.php

```diff
@@ -65,13 +68,16 @@ private static function databaseSchema(string $tablePrefix): Table
             (new Column('propertyname', Type::getType(Types::STRING)))->setLength(255)->setNotnull(true)->setDefault('')
         ]);
 
-        return $table
+        $table
             ->addUniqueIndex(['assetid', 'originalassetid', 'contentstreamid', 'nodeaggregateid', 'origindimensionspacepointhash', 'propertyname'], 'assetperproperty')
             ->addIndex(['assetid'])
             ->addIndex(['originalassetid'])
             ->addIndex(['contentstreamid'])
             ->addIndex(['nodeaggregateid'])
-            ->addIndex(['origindimensionspacepointhash']);
+            ->addIndex(['origindimensionspacepointhash'], options: ['lengths' => [768]]);
```

Comment:
This change is not directly related but a follow-up to #4088 – without that index length limitation, `setup()` will always drop and re-add the column (and the status will reflect that)